canonical-web-and-design/ubuntu.com
# USN nav

## Done

- Added "Notices" to Security sub nav
- Added blank template at /security
- Changed links to usn.u.c around the site to /security/notices

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- In the developer mega nav, click "Security notices"
- See that you are taken to /security/notices (the page will be empty except for site furniture)
- See that "Notices" is highlighted in the sub nav
- Click "Notices" in the footer, see that you are brought back to the same page
- Visit /security and find the "Security notices" link
- Click it, see that you are taken to /security/notices, not usn.u.c


## Issue / Card

Fixes #6325
